### DISCUSSION TITLE
Can we bing back Satellite maps and other maps to the Configurator

### DISCUSSION THREAD

**Discussion by synersignart** [iNavFlight/inav-configurator#1639] [Ideas]
Id wish to bring back sattelite maps and other maps back to the configurators mission control tab . 

I found a Legacy Gmap which is still working
https://github.com/synersignart/GMAP-Legacy-

note: for google use Goggle China Satellite map as this still uses Legacy protocol
 Bing Satellite maps still works

**Comment by MrD-RC** [comment]
If you setup bing maps as the provider, it is a satellite map.

**Comment by DzikuVx** [comment]
Google Maps does not allow vehicle tracking under free licence

**Reply by synersignart** [reply]
ah ok , didnt realize the changes

**Comment by DzikuVx** [comment]
We are not allowed to distribute google maps integration.

Bing Maps allow vehicle tracking but we are not allowed to distribute Configurator with API key. User has to obtain own key and enable bing Maps by entering the key into Configurator. Read the Readme file
